www/index.html

```diff
@@ -67,7 +67,7 @@
             },
             onDeviceReady: function() {
                 document.addEventListener("backbutton", onBackPressed, false);
-                codePush.sync();
+//                codePush.sync();
```

Comment:
Want to suggest we disable it at the moment? seems not reliable, and we have not verify it, it caused problem with QA as they keep test on an outdated version despite you've published